server/chat-plugins/hangman.ts

```diff
@@ -268,26 +268,26 @@ export const commands: ChatCommands = {
 
 		disable(target, room, user) {
 			if (!this.can('gamemanagement', null, room)) return;
-			if (room.hangmanDisabled) {
+			if (room.settings.hangmanDisabled) {
 				return this.errorReply("Hangman is already disabled.");
 			}
-			room.hangmanDisabled = true;
-			if (room.chatRoomData) {
-				room.chatRoomData.hangmanDisabled = true;
-				Rooms.global.writeChatRoomData();
+			room.settings.hangmanDisabled = true;
+			if (room.settings.persistSettings) {
+				room.settings.hangmanDisabled = true;
+				room.saveSettings();
 			}
```

Comment:
```suggestion
			room.settings.hangmanDisabled = true;
			room.saveSettings();
```
You have `room.settings.hangmanDisabled = true` in there twice...